test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -542,8 +542,8 @@ public static IDataView PrepareData(MLContext mlContext, IDataView data)
             // Define the operation code.
             Action<InputRow, OutputRow> mapping = (input, output) => output.Label = input.Income > 50000;
             // Make a custom transformer and transform the data.
-            var transformer = mlContext.Transforms.CustomMappingTransformer(mapping, null);
-            return transformer.Transform(data);
+            var transformer = mlContext.Transforms.CustomMapping(mapping, null);
```

Comment:
`estimator` because this is a how-to doc. :)

Reply:
good point!

---
In reply to: [263484241](https://github.com/dotnet/machinelearning/pull/2876#discussion_r263484241) [](ancestors = 263484241)